README.md

```diff
@@ -324,10 +324,19 @@ Table component supporting sorting and filtering.
 | data (required) | Array | The array of data items, each corresponding to a single potential table row | None
 | filter (optional) | Function | Called with data for a single row. Should return `false` if the row should be filtered out, or `true` otherwise. | None
 | fixed (optional) | Boolean | Whether or not table column widths should be fixed (vs fluid). | False
+| initialPage (optional) | Number | The initial page to be displayed initially. | None
 | initialSortState (optional) | `{columnID: String, direction: Table.sortDirection}` | The initial sort state of the table. | None
+| onPageChange (optional) | Function | Callback function for the displayed page change event. | None
```

Comment:
Does this happen asynchronously as the page change occurs, before, or after it? I could imagine implementing tables that fetch new data on page changes by listening on this function somehow (and backfilling fake data in `data`) if the actual page change blocked on this completing; but idk if that's a pattern we want to prevent altogether.